src/styles/blocks/calendar-block.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  width: 726px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  gap: 1px;
+
+  @each $value in $days-of-mounth {
+    &--month-length-#{$value} {
+      .calendar__day:nth-child(-n + #{$value}) {
+        @include size-day-cells;
```

Comment:
calendar day should have this rule by default, because now if i change modifier to 32 days disappear completely
![image](https://user-images.githubusercontent.com/54936044/232202471-a9eaddde-ad66-4fed-a977-5ec3eeebe028.png)